### PR TITLE
Add event listeners and event.js file

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
In this project, I was required to add some functionality to the application to make it interactive:
- Add an event.js file
- Add event listener to the checkbox
-  Updated array of items in local storage

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import Icon from './enter.png';
 import Icon2 from './icons8-available-updates-96.png';
 import Icon3 from './options.png';
 import Icon4 from './icons8-delete-64.png';
+import delTag from './event.js';
 
 class TDL {
```
Comment (Lameck1) [inline src/index.js:8]: - [ ] Everything is working fine except for the last project requirement which I will quote below 👇🏼

   > Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

Take a look at the screenshot below:

tasks

The `localStorage` is updated just fine and even when the page reloads, the items in the localStorage maintain their state. The problem is with the UI not reflecting the correct state of data from the localStorage i.e. when a task is marked as complete in the localStorage, that should reflect in the UI as well. Look at the UI below:

UI

### Solution
Give your checkbox a `.checked` value assigned to `.completed` value of a task:

```JavaScript
checkBox.checked = listOfItems.completed;
```

Additionally, the strikethrough on the task's description should also be maintained when a task is marked as complete.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @hamid-murambiwa, 

Good job so far!
We however have an issue to address before we can move ahead.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (rloterh) [inline src/index.js:8]: This section needs further improvements.
- The _add_ and _remove_ functionality is **not working as intended**. Kindly ensure to improve upon the following:
- [x] The _Clear all completed_ should be clear selected items, and the form should be able to add items. 

Kindly correct the following errors (as seen in the browser console below) of your buttons button clicks implementations:

image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Hamid, 

You did great on this project, and for making the changes requested by the previous reviewer! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments above, made under this review. Kindly take your time to correctly review those issues outlined in the previous code reviews, and please check thoroughly to see that the changes are working as intended._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider the comments marked **[OPTIONAL]** under this review:

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (hamid-murambiwa) [reply src/index.js:8]: the add and the clear all completed functions are not required for this project, so I will add them on the next branch.

**Review by dasileker — APPROVED** [review]
## **Hi @hamid-murambiwa,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**